codelists/actions.py

```diff
@@ -573,7 +575,11 @@ def export_to_builder(*, version, author, coding_system_database_alias):
     # the builder frontend cannot deal with a CodeObj with status ?  if any of its
     # ancestors are included or excluded.  We will have to deal with this soon but for
     # now fail loudly.
-    assert not draft.code_objs.filter(status="?").exists()
+    if not ignore_unknown_statuses:
+        assert not draft.code_objs.filter(
```

Comment:
Rather than an `assert` here, how easy it would it be to raise some kind of error we could bubble up to the user? If the answer is "not easy" or this happens infrequently enough that it's just not worth it then fine. I'm just thinking of things which might make our life easier next time this happens.

Reply:
I think it's probably easy enough, and I expect it to happen fairly often when a user tries to make a new draft from an old-ish codelist version (although I'm not entirely sure how often _that_ happens).  It'd be better than the obscure 500 though